.craft.yml

```diff
@@ -1,10 +1,11 @@
-github:
-  owner: getsentry
-  repo: sentry-toolbar
+minVersion: '0.23.1'
```

Comment:
what's this version?

Reply:
i copied it in from the docs

idk even know what the latest version is tho, can remove.